prow/cmd/README.md

```diff
@@ -0,0 +1,46 @@
+# Prow Images
+
+This directory includes a sub directory for every Prow component and is where all binary and container images are built. You can find the `main.go` files here in addition to the `BUILD` files that contain Bazel rules for building binary and container images.
+
+## Cluster Components
+
+Prow has a microservice architecture implemented as a collection of container images that run as Kubernetes deployments. A brief description of each service component is provided here.
+
+#### Core Components
+
+* [`hook`](/prow/cmd/hook) is the most important piece. It is a stateless server that listens for GitHub webhooks and dispatches them to the appropriate plugins. Hook's plugins are used to trigger jobs, implement 'slash' commands, post to Slack, and more. See the [`prow/plugins`](/prow/plugins/) directory for more information on plugins.
+* [`plank`](/prow/cmd/plank) is the controller that manages the job execution and lifecycle for jobs that run in k8s pods.
+* [`deck`](/prow/cmd/deck) presents [a nice view](https://prow.k8s.io/) of recent jobs, some help information, the status of merge automation, and a dashboard for PR authors.
```

Comment:
maybe also link to plugins help/tide/pr status?

Reply:
Done.